keyboards/4pplet/waffling60/info.json

```diff
@@ -1,80 +0,0 @@
-{
-    "manufacturer": "4pplet",
-    "url": "https://github.com/4pplet/waffling60",
-    "maintainer": "4pplet",
-    "usb": {
-        "vid": "0x4444"
```

Comment:
Looks like moving/removing this breaks the previous revisions.

Reply:
My mistake! Should be fixed now ^^